src/main/java/racingcar/Cars.java

```diff
@@ -0,0 +1,68 @@
+package racingcar;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private static final String CAR_NAME_DELIMITER = ",";
+    public static final int DEFAULT_POSITION = 0;
+
+    private final List<Car> cars;
+
+    public Cars(String s) {
+        this(Arrays.asList(s.split(CAR_NAME_DELIMITER)));
+    }
+
+    public Cars(List<String> carNames) {
+        validateNotDuplicated(carNames);
+        this.cars = carNames.stream().map(Car::new).collect(Collectors.toList());
```

Comment:
stream을 사용하실 때는 한 줄의 하나의 함수만 호출하면  가독성을 높일 수 있어요. 😉

```suggestion
        this.cars = carNames.stream()
        .map(Car::new)
        .collect(Collectors.toList());
```